app/views/spotlight/exhibits/_exhibit_card_back.html.erb

```diff
@@ -18,6 +18,15 @@
   </p>
 <% end %>
 
+<%
+  # The inclusion of the hidden `<div>` addresses an accessibility issue where a link label should not be used with multiple link
+  # locations.  In this case, the label is 'Visit exhibit' and it is used for all links leading to the various exhibits.
+  #
+  # Using just the `exhibit.id` is not sufficient to create a unique id as this code is used in each tab (i.e. Published exhibits,
+  # Unpublished exhibits, and Your exhibits).  To ensure the ID is unique across tabs, it is appended with a random hex string.
+  label_id = "visit-exhibit-#{exhibit.id}-#{SecureRandom.hex(8)}"
+%>
+<div class="hidden" id="<%= label_id %>"><%= "#{t(:'.visit_exhibit')} #{exhibit.title}" %></div>
```

Comment:
I think we would want to pass the `exhibit.title` in as an argument to the `i18n` given that different languages may not use the same sentence structure that we do in english.  However; at that point, I wonder if makes sense to just have the i18n string passed to the `link_to` pass the exhibit title in (since we're using the same key here anyway) and have it produce HTML with the exhibit title in a `.sr-only` span.  This way the link itself would read as "Visit exhibit" to sighted users but "Visit exhibit {title of exhibit}" to screen reader users w/o introducing additional elements and aria attributes.